main/main.cpp

```diff
@@ -133,10 +134,13 @@ extern "C" void app_main()
     err = radio::init();
     if (err != ESP_OK) {
         ESP_LOGE(TAG, "Couldn't initialize radio, err = %s",
-                 esp_err_to_name(err));
+            esp_err_to_name(err));
         return;
     }
 
+
+    Vaina::getInstance();
+
```

Comment:
This can be removed, when `getInstance` gets called where it's needed it will get constructed at the given time, that's called [lazy initialization](https://en.wikipedia.org/wiki/Lazy_initialization). i.e.: it's created only when it's needed :wink: